Source/Blazorise/Components/Tooltip/Tooltip.razor.cs

```diff
@@ -305,7 +314,7 @@ public TooltipTrigger Trigger
     /// <summary>
     /// Specifies the delay in ms once a trigger event is fired before a Tooltip shows or hides.
     /// </summary>
-    [Parameter] public (int Show, int Hide) Delay { get; set; } = (0, 0);
+    [Parameter] public (int Show, int Hide)? Delay { get; set; }
```

Comment:
What about `ShowDelay` and `HideDelay` as two separate parameters? 
I still don't like how we changed our mind about using values tuples, I would almost certainly say we don't have any other api with value tuples. I like them more for internal stuff and less for public stuff.
I'd just keep it consistent and use the same as we've been doing.

I honestly think the two separate params are more clear as a public api. I don't mind either anyway.

Reply:
The more I'm thinking, you're right. It is better not to use a tuple and to have a separate parameters instead. Going to change it now.